src/bundle/Resources/public/js/scripts/edit.header.js

```diff
@@ -50,4 +50,5 @@
 
     contentNode.addEventListener('scroll', fitHeader, false);
     controlZIndex(headerNode);
+    controlZIndex(headerBottomRowNode);
```

Comment:
It is necessary to add a z-index control because the header in the slim version has z-index set to 1 and the modal share dialog is is covered with a backdrop